Modules/Segmentation/LevelSets/include/itkVectorThresholdSegmentationLevelSetFunction.hxx

```diff
@@ -21,7 +21,6 @@
 #include "itkImageRegionIterator.h"
 #include "itkGradientAnisotropicDiffusionImageFilter.h"
 #include "itkLaplacianImageFilter.h"
-#include "itkImageFileWriter.h"
```

Comment:
This `#include "itkImageFileWriter.h"` appears to be introduced with commit e795b8be95b295f4ce38b9cea445b3edadb47db6, "Vector version of the ThresholdSegmentationLevelSet method", Luis Ibanez, Dec 18, 2003. But without actually being used! Looking at
https://github.com/InsightSoftwareConsortium/ITK/blob/e795b8be95b295f4ce38b9cea445b3edadb47db6/Code/BasicFilters/itkVectorThresholdSegmentationLevelSetFunction.txx